contracts/test/shared/errors.t.sol

```diff
@@ -27,7 +27,8 @@ contract Errors is DSTest {
     OnlyRenounceRolesForSelf,
     ContractIsNotInitializing,
     ContractIsInitializing,
-    ContractAlreasyInitialized
+    ContractAlreasyInitialized,
+    OnlyAdmin
```

Comment:
for a error type to be available you need to define it in this enum, like you did, but then link it to the actual error string in the constructor function, but in your case the error is already defined as "CallerNotAdmin"